etna/models/sarimax.py

```diff
@@ -274,7 +274,7 @@ def predict(self, df: pd.DataFrame, prediction_interval: bool, quantiles: Sequen
                 y_pred[f"mean_{quantile:.4g}"] = series
         else:
             forecast = self._result.get_prediction(
-                start=df["timestamp"].min(), end=df["timestamp"].max(), dynamic=True, exog=exog_future
+                start=df["timestamp"].min(), end=df["timestamp"].max(), dynamic=False, exog=exog_future
```

Comment:
Let's make in the other PR

Reply:
I created a new issue that will be made later #811.